### DISCUSSION TITLE
Water meter housing (DIEHL water meter MOM2020 | M20 1422)

### DISCUSSION THREAD

**Discussion by nuki47** [jomjol/AI-on-the-edge-device#2782] [Q&A]
I have a DIEHL water meter (MOM2020 | M20 1422 ), but unfortunately the recommended AI-on-the-edge housing [1] doesn't really fit. Do you have a more suitable recommendation?

![1728e904c040a34d07b61b6b6ea86e972adedcf1](https://github.com/jomjol/AI-on-the-edge-device/assets/75117725/64d2c404-d774-413a-97c7-3286f04e7192)

[1] https://www.thingiverse.com/thing:4573481

**Comment by caco3** [comment]
You will have to design your own housing. Search the discussions here, there are many cool ideas, also without using 3D printing
